core/src/test/java/com/redhat/lightblue/client/response/TestLightblueResponse.java

```diff
@@ -25,6 +25,17 @@ public void setUp() throws Exception {
         testResponse = new LightblueResponse(initialResponseText);
     }
 
+    @Test(expected = RuntimeException.class)
```

Comment:
No, it should be a RuntimeException. This test is for the constructor which can throw a RuntimeException if an IOException occurs.